src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -220,9 +231,10 @@ export default class Pilot {
      * @for Pilot
      * @method maintainSpeed
      * @param {Number} speed - the speed to maintain, in knots
+     * @param {AircraftModel} aircraftModel
      * @return {Array} [success of operation, readback]
      */
-    maintainSpeed(currentSpeed, speed) {
+    maintainSpeed(currentSpeed, speed, aircraftModel) {
         const instruction = radio_trend('speed', currentSpeed, speed);
```

Comment:
We can get rid of one of the speed params here since we also send the `aircraftModel`